src/server/index.js

```diff
@@ -65,8 +65,11 @@ function Server({ games, db }) {
           const roomClients = roomInfo.get(gameID);
           for (const client of roomClients.values()) {
             const playerID = clientInfo.get(client).playerID;
+
+            let newCtx = { ...state.ctx, seed: undefined };
```

Comment:
just inline this below?

Reply:
`newCtx` is used twice, since calling into `playerView` also needs the `ctx`. I've changed the code to use the old `state.ctx` to use in `playerView`, and inlined the code.